### PR TITLE
📖  Updated the community links

### PR DESCRIPTION
**Pull request by naveensrinivasan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

     Updated the community links to be more accessible. Took the idea from https://github.com/tektoncd/community/blob/main/working-groups.md#chains

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Review by azeemshaikh38 — APPROVED** [review]
LGTM. Will leave it to @olivekl for the final approval.

**Comment by naveensrinivasan** [comment]
> LGTM. Will leave it to @olivekl for the final approval.

Thanks!

**Comment by naveensrinivasan** [comment]
@laurentsimon  I included your name in this. I completely forgot to get your approval. Can you please review it?

File: README.md
Comment (laurentsimon) [inline README.md:400]: is it standard to provide easy-to-crawl/parse meeting links this way? No fear of zoom-bombing or the like? I don't think we can do anything about it, but if you come across a solution, let's use it in the future

**Review by laurentsimon — APPROVED** [review]
LGTM, thanks!

**Comment by olivekl** [comment]
My only concern would be including the Zoom link, which Laurent mentioned as well. Otherwise, everything LGTM. Thanks for these improvements!

**Comment by naveensrinivasan** [comment]
> My only concern would be including the Zoom link, which Laurent mentioned as well. Otherwise, everything LGTM. Thanks for these improvements!

I updated the Zoom link title not to have the actual link. Thanks

**Comment by laurentsimon** [comment]
> > My only concern would be including the Zoom link, which Laurent mentioned as well. Otherwise, everything LGTM. Thanks for these improvements!
> 
> I updated the Zoom link title not to have the actual link. Thanks

Maybe we could say "See calendar invite below"

**Pull request merged**
